src/fmu/sumo/explorer/_case.py

```diff
@@ -48,17 +48,16 @@ def get_object_types(self) -> Dict[str, int]:
 
         return self.utils.map_buckets(buckets)
 
-    def get_summary_blob_ids(self, **kwargs) -> Dict[str, str]:
-        """Gets blob_ids for summary data aggregated per vector.
 
+    def get_summary_object_ids(self, **kwargs) -> Dict[str, str]:
+        """Gets blob_ids for summary data aggregated per vector
         args kwargs (dict): various keyword arguments
         returns dictionary with vector name as key, value object id
         """
-        return get_object_blob_ids(
-            self, data_type="table", content="timeseries", **kwargs
-        )
+        return get_object_ids(self, data_type="table", content="timeseries", **kwargs)
+
+    def get_object_ids(
```

Comment:
Is this rename breaking the API? If so, should the old name still be valid? E.g. turn it into an alias for the current method + a deprecation warning?